tests/functional/features/chart_submitted_by_unauthorized_user.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Chart submitted by unauthorized user
+    Partners or redhat associates can not submit charts if they are not in the OWNERS file
```

Comment:
Might be better to specify which OWNERS file, since we have an OWNERS file at root of this repository.
```suggestion
    Partners or redhat associates can not submit charts if they are not in the OWNERS file of the chart
```
or something similar